synoptic/src/synoptic/tests/SynopticTest.java

```diff
@@ -57,6 +58,30 @@ public void setUp() throws ParseException {
         super.setUp();
     }
 
+    /*
+     * Due to differences between command line and IDE usage, checks the path
+     * used and adjusts it accordingly.
+     */
+    public static String getTestPath(String path) throws Exception {
+
+        // original path passed in
```

Comment:
Nit: Capitalize to keep comments consistent with the rest of the codebase ("**O**riginal path passed in"). Same for other comments